test/test-buildSrcSet.js

```diff
@@ -953,6 +953,20 @@ describe('SrcSet Builder:', function describeSuite() {
                     }, Error);
                 });
             });
+
+            describe('with widthTolerance, minWidth, and maxWidth set to promote an edge case', function describeSuite() {
```

Comment:
But 0.0999 isn't some magic number, nor are 0.09 and 0.10. There will be more failing cases at other values of `widthTolerance`, or for `0.09` or `0.10` for other values of `minWidth` and `maxWidth`. The test isn't really about those specific numbers; it's about coaxing out an edge case which reveals a bug.

Reply:
This may just be a difference in how we like to describe tests. I like to describe the bug I'm trying to highlight, if possible, whereas perhaps you like to describe particular error-prone circumstances.

I'll change the name of it to what you suggested if you confirm.

Reply:
This is a hard test case to find a good name for. Overall I'm in agreement with @tremby here. I don't think we should hard-code a value in the test case name, because it doesn't capture the purpose of the test. I think we could improve a bit (as already mentioned) what an "edge case" is, e.g. something like "with widthTolerance, minWidth and maxWidth values that have caused duplicate values in the past"

Reply:
I've changed some names accordingly.